src/System.Private.Xml/tests/Xslt/XslTransformApi/CXslTransform.cs

```diff
@@ -498,17 +488,10 @@ public void XmlResolver3(InputType inputType, ReaderType readerType, TransformTy
         [InlineData(InputType.Navigator, ReaderType.XmlValidatingReader, TransformType.Writer, DocType.XPathDocument)]
         [InlineData(InputType.Navigator, ReaderType.XmlValidatingReader, TransformType.Stream, DocType.XPathDocument)]
         [InlineData(InputType.Navigator, ReaderType.XmlValidatingReader, TransformType.TextWriter, DocType.XPathDocument)]
-        [Theory(Skip = "When style sheet URI = Intranet zone, XmlSecureResolver does not resolve document function")]
-        public void XmlResolver7(InputType inputType, ReaderType readerType, TransformType transformType, DocType docType)
+        [Theory(Skip = "Absolute path is hard coded, it needs to be set dynamically")]
```

Comment:
This one I am planning to add to this on the PR (fixes #21201), I just wanted to get some early feedback on the changes so far.